opm/simulators/timestepping/AdaptiveTimeSteppingEbos.hpp

```diff
@@ -322,6 +322,7 @@ namespace Opm {
                         Opm::time::StopWatch perfTimer;
                         perfTimer.start();
 
+                        ebosProblem.setForcedNextStepSize(dtEstimate);
                         ebosProblem.writeOutput(/*isSubStep=*/true);
```

Comment:
This line also needs to be added after
https://github.com/OPM/opm-simulators/blob/657c45be92f23215f9bc78000a0dcad9dfa2c66d/opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp#L291

Then Jenkins is happy

Reply:
Oops, thanks for finding this. actually, the machinery for forcing the next time step size was already present but simply not used by the new timestep proposal machinery in ebos. fixed in the ewoms PR. (BTW: this only has only ever worked with flow's adaptive time stepper.)

Reply:
Great. That means one hack less.